pkg/publish/default.go

```diff
@@ -47,8 +47,8 @@ type defalt struct {
 	insecure  bool
 }
 
-// Option is a functional option for NewDefault.
-type Option func(*defaultOpener) error
+// DefaultOption is a functional option for NewDefault.
+type DefaultOption func(*defaultOpener) error
```

Comment:
This change will probably cause some pain to consumers of this package as a library. Can we keep this as `Option`?

Reply:
Reverted.